app/controllers/sessions_controller.rb

```diff
@@ -6,7 +6,7 @@ def new
 
   def create
     user = User.find_by(email: params[:email])
-    flash[:success] = "Welcome, #{user.name}!"
+    flash[:success] = "Logged in as #{user.name}"
```

Comment:
I like this wording better. I know I moved this flash message down until after the password verification in order to finish user story 14, so we will need to check that both changes survive the merge :).

Reply:
I changed this message based on the user story. Yeah we may need to come back to this part later